dwitter/serializers.py

```diff
@@ -6,35 +6,53 @@
 
 
 class UserSerializer(serializers.ModelSerializer):
+    link = serializers.SerializerMethodField()
+    date_joined = serializers.DateTimeField(format="%Y-%m-%d")
+
     class Meta:
         model = User
-        fields = ('pk', 'username')
+        fields = ('username', 'date_joined', 'link')
+
+    def get_link(self, obj):
+        return 'https://www.dwitter.net/u/%s' % obj.username
```

Comment:
I was using the urlresolver in the insert_magic_links filter, just to be safe, but hard-coding the URL should be fine for all intents and purposes :)